internal/dcache/rpc/server/handler.go

```diff
@@ -133,7 +133,16 @@ type syncInfo struct {
 	sourceRVName string // source RV name for syncing this MV
 }
 
+var handler *ChunkServiceHandler
+
+// NewChunkServiceHandler creates a new ChunkServiceHandler instance
+// This is a singleton instance and is created only once.
+// Subsequent calls to this function will return the same instance.
 func NewChunkServiceHandler() *ChunkServiceHandler {
+	if handler != nil {
+		return handler
```

Comment:
if we don't have a usecase of calling this more than once, let's assert here

Reply:
We can call this more than once if we need information about the reserved space in an RV. I have made this singleton, so that if cluster manager wants, it can refer the same instance and get the required information about the RV

Reply:
One more information it can get the totalChunkBytes inside an MV when making JoinMV call